### PR TITLE
setting headers with id when writing topics

### PR DESCRIPTION
**Pull request by Spartakovic**

### CODE REVIEW

File: src/main/java/org/atlasapi/query/v2/TopicWriteController.java [hunk 1]
```diff
@@ -96,7 +101,13 @@ public Void writeContent(HttpServletRequest req, HttpServletResponse resp) {
             log.error("Error reading input for request " + req.getRequestURL(), e);
             return error(resp, HttpStatusCode.SERVER_ERROR.code());
         }
-        
+
+        String hostName = Configurer.get("local.host.name").get();
+        resp.setHeader(
+                HttpHeaders.LOCATION,
+                hostName
+                        + "/3.0/content.json?id="
```
Comment (tfm) [inline src/main/java/org/atlasapi/query/v2/TopicWriteController.java]: Don't you mean topic?

File: src/main/java/org/atlasapi/query/v2/TopicWriteController.java [hunk 2]
```diff
@@ -45,6 +49,7 @@
     private final ApplicationConfigurationFetcher appConfigFetcher;
     private final TopicStore store;
     private final ModelReader reader;
+    private static final NumberToShortStringCodec codec = SubstitutionTableNumberCodec.lowerCaseOnly();
```
Comment (tfm) [inline src/main/java/org/atlasapi/query/v2/TopicWriteController.java]: Do topics use the uppercase or lowercase-only codec? Please double-check.

Comment (Spartakovic) [inline src/main/java/org/atlasapi/query/v2/TopicWriteController.java]: they use lowercase codec

Comment (Spartakovic) [inline src/main/java/org/atlasapi/query/v2/TopicWriteController.java]: yes they use lowercase

**Comment by Dalamar42** [comment]
LGTM

**Pull request merged**